setup/project_template/odoo/addons/project_template

```diff
@@ -0,0 +1 @@
+../../../../project_template
```

Comment:
May this file be removed from the commit?

Reply:
I agree